test/cets_SUITE.erl

```diff
@@ -73,53 +121,11 @@ cases() ->
         insert_serial_works_when_leader_is_back,
         insert_serial_blocks_when_leader_is_not_back,
         leader_is_the_same_in_metadata_after_join,
-        join_with_the_same_pid,
-        join_ref_is_same_after_join,
-        join_fails_because_server_process_not_found,
-        join_fails_because_server_process_not_found_before_get_pids,
-        join_fails_before_send_dump,
-        join_fails_before_send_dump_and_there_are_pending_remote_ops,
-        send_dump_fails_during_join_because_receiver_exits,
-        join_fails_in_check_fully_connected,
-        join_fails_because_join_refs_do_not_match_for_nodes_in_segment,
-        join_fails_because_pids_do_not_match_for_nodes_in_segment,
-        join_fails_because_servers_overlap,
-        remote_ops_are_ignored_if_join_ref_does_not_match,
-        join_retried_if_lock_is_busy,
         send_dump_contains_already_added_servers,
         servers_remove_each_other_if_join_refs_do_not_match_after_unpause,
```

Comment:
Just to be sure, is this not a testcase that could go to `cets_join_SUITE.erl`? It's the only one using the `set_join_ref/2` helper outside of this suite.